app/config/config.yml

```diff
@@ -77,16 +77,7 @@ doctrine:
         # them from the app/config/parameters.yml file. The reason is that config.yml
         # stores options that change the application behavior and parameters.yml
         # stores options that change from one server to another
-        driver:   "%database_driver%"
-        host:     "%database_host%"
-        port:     "%database_port%"
-        dbname:   "%database_name%"
-        user:     "%database_user%"
-        password: "%database_password%"
-        charset:  UTF8
-        # if using pdo_sqlite as your database driver, add the path in parameters.yml
-        # e.g. database_path: "%kernel.root_dir%/data/data.db3"
-        path:     "%database_path%"
+        url:      "%database_url%"
```

Comment:
Even if configuring "just" the URL seems better at first (and I understand why this is needed for Heroku), it breaks how people configure a Symfony app right now: config.yml can contain settings for all environments (like the driver), and config_dev.yml and config_prod.yml (or their parameters.yml counterparts) can "just" change the moving parts like the username or the password.